examples/src/main/java/com/flipkart/gjex/examples/helloworld/filter/AuthGrpcFilter.java

```diff
@@ -15,27 +15,25 @@
  */
 package com.flipkart.gjex.examples.helloworld.filter;
 
-import javax.inject.Named;
-
-import com.flipkart.gjex.core.filter.Filter;
-import com.flipkart.gjex.core.filter.ServerRequestParams;
-import com.flipkart.gjex.core.logging.Logging;
-
+import com.flipkart.gjex.core.filter.RequestParams;
+import com.flipkart.gjex.core.filter.grpc.GrpcFilter;
 import io.grpc.Metadata;
 import io.grpc.Status;
 import io.grpc.StatusRuntimeException;
 import io.grpc.examples.helloworld.HelloReply;
 import io.grpc.examples.helloworld.HelloRequest;
 
+import javax.inject.Named;
+
 /**
- * An implementation of the {@link Filter} interface as example that performs naive authentication based on
+ * An implementation of the {@link GrpcFilter} interface as example that performs naive authentication based on
  * information contained in the Request headers
  *
  * @author regu.b
  *
  */
-@Named("AuthFilter")
-public class AuthFilter implements Filter<HelloRequest, HelloReply>, Logging {
+@Named("AuthGrpcFilter")
+public class AuthGrpcFilter extends GrpcFilter<HelloRequest, HelloReply> {
```

Comment:
This can remain `AuthFilter`

Lets not change the grpc filter names. Gjex is major for grpc and http is provided for migration period. In long term i don't see the need for having this explicit name in filter.s

Reply:
done